modules/apps/blogs/blogs-web/src/main/java/com/liferay/blogs/web/internal/upload/ImageBlogsUploadFileEntryHandler.java

```diff
@@ -70,19 +72,24 @@ public FileEntry upload(UploadPortletRequest uploadPortletRequest)
 			ActionKeys.ADD_ENTRY);
 
 		String fileName = uploadPortletRequest.getFileName(_PARAMETER_NAME);
-		String contentType = uploadPortletRequest.getContentType(
-			_PARAMETER_NAME);
 
-		_validateFile(
-			fileName, contentType,
-			uploadPortletRequest.getSize(_PARAMETER_NAME));
+		if (Validator.isNotNull(fileName)) {
```

Comment:
@adolfopa What do you think about reusing the UploadFileEntryHandler to upload the modified image from the selector?

Is the same action, uno from file, one form blob image.
This way we do not have to make another module and different handlers for the different modules.
We reuse the error handlers and in some cases code.

let me know what you think about it.